setup.py

```diff
@@ -53,17 +53,11 @@
 ]
 
 install_requires = [
-    "checksumdir>=1.1.4,<1.2",
     "graphviz>=0.12",  # FIXME needed only if yadage visuale=True.
-    "mock>=3.0,<4",
     "pydot>=1.0.29",  # FIXME needed only if yadage visuale=True.
     "pydotplus>=2.0.2",  # FIXME needed only if yadage visuale=True.
-    "adage==0.10.1",  # FIXME remove once yadage>=0.20.2 is compatible.
-    "yadage==0.20.1",  # FIXME remove once yadage>=0.20.2 is compatible.
-    "yadage-schemas==0.10.6",  # FIXME remove once yadage>=0.20.2 is compatible.
```

Comment:
Removed because `yadage` v0.20.2 was yanked from PyPI

Reply:
Beware, this could be dangerous. E.g. What if yadage 0.20.3 is released and is not compatible with how we implemented r-w-e-yadage?

We could do this _if_ we have a strict yadage version pin in the client (as for snakemake or cwl), but we don't, since we would like to give people a chance to use "any yadage version" locally. 

I would therefore keep the pin on the server side to be fully sure that we are using a Yadage version that we know we support well.

Reply:
There are already some constraints in `reana-commons`, see [here](https://github.com/reanahub/reana-commons/blob/eb3d72d1d0f9c3e660f6bae11c020408077797dc/setup.py#L40):

```
 "yadage": ["adage~=0.10.1", "yadage~=0.20.1", "yadage-schemas~=0.10.6"],
```

These pins however permit to use an hypothetical v0.20.3. Should I make the pin more restrictive in `reana-commons` or should I add them here? (the first option would require a new release)

Reply:
That was the problem with 0.20.2 in the past, the release "broke" server, even though it promised compatibility (`~=`). So, to be on the safe side, it's good to pin Yadage to what works. And we should do it on the server-side, because some persons used other Yadage versions on the client side. This will avoid having to do a `reana-commons` release. So I would pin just the adage/yadage/schemas/packtivity combo on `r-server`, and that should do.

BTW after we shall move the `create`/`validate` functionality from the client-side to the server-side, then we shall accept any Yadage version on the client, and ditto for CWL and Snakemake. From this point of view, accepting any compatible Yadage version on the client is already a move towards the future, so to speak. (And it's already done via `~=`, so let's keep it in the `r-commons`.) CWL and Snakemake still use `==` there, but will follow once we shall not need them on the client side as well.